src/main/webapp/assets/modules/user-type/userTypeController.js

```diff
@@ -1,3 +1,3 @@
-techlooper.controller("userTypeController", function ($scope) {
-
+techlooper.controller("userTypeController", function ($scope, $rootScope, securityService) {
```

Comment:
Line is too long.<br>'techlooper' is not defined.<br>'securityService' is defined but never used.<br>'$rootScope' is defined but never used.<br>'$scope' is defined but never used.